tests/qat/test_readout_mitigation.py

```diff
@@ -328,8 +328,14 @@ def test_prepare_bitstring_and_measure(
         q0_ro_fidelity_1=0.7,
         q1_ro_fidelity_0=0.9,
         q1_ro_fidelity_1=0.6,
+        threshold=0.051,
     ):
         assert len(bitstring) == 2
+        # Fix the seed for the experiment
+        np.random.seed(0)
+        seed(0)
```

Comment:
why are we seeding using both the python core library and numpy ? isn't numpy enough ?
For repurposing the test, I suggest add a follow up ticket and involve Bryn as he originally put up the random sim test

Reply:
> why are we seeding using both the python core library and numpy ? isn't numpy enough ? For repurposing the test, I suggest add a follow up ticket and involve Bryn as he originally put up the random sim test

RealtimeChipSimEngine uses random() from the core library - had to seed both to have a certain result